### ISSUE TITLE
Excel considers workbook corrupt when setting range of cells to an R1C1 formula that references another sheet

### ISSUE THREAD

**Issue by Thealexbarney** [EPPlusSoftware/EPPlus#253]
Setting the equivalent non-R1C1 formula works. I'm guessing because the R1C1 formula uses a shared formula while the non-R1C1 formula doesn't.
```cs
using (var package = new ExcelPackage(new FileInfo("test.xlsx")))
{
    ExcelWorksheet ws = package.Workbook.Worksheets.Add("Sheet1");
    ExcelWorksheet ws2 = package.Workbook.Worksheets.Add("Sheet2");
    ws2.Cells[1, 3, 4, 3].Value = 2;
    ws2.Cells[2, 3].Value = 1;
    ws2.Cells[4, 3].Value = 4;
    ws.Cells[1, 2].Value = 1;
    ws.Cells[2, 2].Value = 2;
    ws.Cells[3, 2].Value = 3;
    ws.Cells[4, 2].Value = 4; 
    ws.Cells[1, 1, 4, 1].FormulaR1C1 = "COUNTIF(Sheet2!R1C3:R4C3, RC2)";
    // ws.Cells[1, 1, 4, 1].Formula = "COUNTIF(Sheet2!$C$1:$C$4, $B1)"; // Works

    package.Save();
}
```

**Comment by JanKallman** [comment]
Thanks for reporting it. Fix applied. Feel free to test it using our develop nuget feed - https://ci.appveyor.com/nuget/epplus.

**Comment by JanKallman** [comment]
Fixed in EPPlus 5.5
